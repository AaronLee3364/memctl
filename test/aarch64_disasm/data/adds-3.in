fffffff0073ae26c ab2a4109
