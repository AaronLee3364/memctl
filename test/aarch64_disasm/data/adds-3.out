fffffff0073ae26c  ADDS    X9, X8, W10, UXTW
